services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/HankeControllerTest.kt

```diff
@@ -90,14 +91,12 @@ class HankeControllerTest {
         Assertions.assertThatExceptionOfType(ConstraintViolationException::class.java).isThrownBy {
             hankeController.updateHanke(partialHanke, "id123")
         }.withMessageContaining("updateHanke.hanke.nimi: " + HankeError.HAI1002.toString())
```

Comment:
We lost now in this test the fact that we want to get too elements' validation error in one go :(  Why do we allow empty createdBy? It should always be there? There is no other way for us to know who is the user creating the object than bringing it here?

Reply:
We can't trust createdBy in future, and have no use for it at the moment as a temporary way of testing (we have no different users at all).. As soon as we add the basic users, and configure spring(-security) appropriately, the user information comes through that path (not in the request JSON).